src/utils.ts

```diff
@@ -41,68 +44,77 @@ export const collectCopyTargets = async (
   structured: boolean,
   silent: boolean
 ) => {
-  const copyTargets: SimpleTarget[] = []
+  const copyTargets: SimpleTarget[][] = await pMap(
```

Comment:
I guess the outer `pMap` is not needed. `fastglob` already runs concurrently internally and I don't expect the `targets` array to be long.

But if you have many `targets`, it might be better to use `stream` (from fast-glob) + `pMapIterable` (from p-map) as that will fill up the concurrency in some cases. Would you try this out if you have many `targets`? If not, I'm also fine with removing the outer `pMap`.

Reply:
Thanks, I don't feel the number of the targets would be a big bottleneck for common use cases, targets won't be that many since globs, filters and renames would reduce the need. I'll remove it in this PR